### DISCUSSION TITLE
NEW model Vicuna13B-v1.1-8bit-128g

### DISCUSSION THREAD

**Discussion by Felixqian4160** [oobabooga/text-generation-webui#1472] [General]
https://huggingface.co/Thireus/Vicuna13B-v1.1-8bit-128g

**Comment by Ph0rk0z** [comment]
You don't have a problem with 8bit GPTQ causing problems with any context at all?

https://github.com/qwopqwop200/GPTQ-for-LLaMa/issues/205

**Comment by SvetZitrka** [comment]
I have a problem with this model. When I load the model I get an error

`Traceback (most recent call last):
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\transformers\modeling_utils.py”, line 442, in load_state_dict
return torch.load(checkpoint_file, map_location=“cpu”)
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\torch\[serialization.py](http://serialization.py/)”, line 791, in load
with _open_file_like(f, ‘rb’) as opened_file:
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\torch\[serialization.py](http://serialization.py/)”, line 271, in _open_file_like
return _open_file(name_or_buffer, mode)
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\torch\[serialization.py](http://serialization.py/)”, line 252, in init
super().init(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: ‘models\Thireus_Vicuna13B-v1.1-8bit-128g\pytorch_model-00001-of-00003.bin’

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File “E:\AI\oobabooga\text-generation-webui\[server.py](http://server.py/)”, line 101, in load_model_wrapper
shared.model, shared.tokenizer = load_model(shared.model_name)
File “E:\AI\oobabooga\text-generation-webui\modules\[models.py](http://models.py/)”, line 207, in load_model
model = LoaderClass.from_pretrained(checkpoint, **params)
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\transformers\models\auto\auto_factory.py”, line 471, in from_pretrained
return model_class.from_pretrained(
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\transformers\modeling_utils.py”, line 2795, in from_pretrained
) = cls._load_pretrained_model(
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\transformers\modeling_utils.py”, line 3109, in _load_pretrained_model
state_dict = load_state_dict(shard_file)
File “E:\AI\oobabooga\installer_files\env\lib\site-packages\transformers\modeling_utils.py”, line 445, in load_state_dict
with open(checkpoint_file) as f:
FileNotFoundError: [Errno 2] No such file or directory: ‘models\Thireus_Vicuna13B-v1.1-8bit-128g\pytorch_model-00001-of-00003.bin’`

And

`Loading Thireus_Vicuna13B-v1.1-8bit-128g...
Auto-assiging --gpu-memory 23 for your GPU to try to prevent out-of-memory errors.
You can manually set other values.
Loading checkpoint shards:   0%|                                                                 | 0/3 [00:00<?, ?it/s]`

**Reply by Ph0rk0z** [reply]
It is not loading with GPTQ.. --wbits 8 not in your command line?

**Reply by Tom-Neverwinter** [reply]
that solves one item then it just makes gibberish.

**Reply by Ph0rk0z** [reply]
try with no context at all and you will get something back from it.

**Reply by Tom-Neverwinter** [reply]
didnt help. I am gussing this may be an issue with AMD users running windows

I needed to remove one version of cuda 11.2 and replaced with 11.8 [tesla m40] if you have a modern card use 12.1

I also completely wiped and reinstalled using the one click installer as I appear to have missed dependancies

**Reply by SvetZitrka** [reply]
I have Intel

**Reply by Ph0rk0z** [reply]
Intel has torch support at all? In any case, this is broken in GPTQ for everyone.
